app/views/posts/show.html.erb

```diff
@@ -1,28 +1,19 @@
-<h2>Here is the details of given post for a given user and its comments</h2>
+<h1>Here is the details of given post for a given user and its comments</h1>
 
-<article>
-  <h3>Post 1 by User 3</h3>
-  <p>Comments:10, likes:2</p>
-  <p>
-    Lorem ipsum dolor sit amet, consectetur adipiscing elit. sed do eiusmod tempor
-    incididunt ut labore et dolore magna aliqua. ut enim ad minim veniam, quis
-    nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
-
-    Lorem ipsum dolor sit amet, consectetur adipiscing elit. sed do eiusmod tempor
-    incididunt ut labore et dolore magna aliqua. ut enim ad minim veniam, quis
-    nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
-  </p>
+<article class = 'user det'>
+  <header class='post-details'>
+    <h2>Post: <%= @post.title %> by <%= @post.author.name %></h2>
+    <p class='cmts-likes'>
+      <span>Comments: <%= @post.comments_counter || 0 %></span>, 
```

Comment:
The comments counter is not working on this page. As you can see, there are 8 comments on this post, still the comments counter is displaying 0. Kindly fix this issue.
![image](https://github.com/LarryIVC/larrys-blog/assets/33268927/fa96c421-121f-4661-910f-1eca7442464c)

Reply:
Fixed